### PR TITLE
fix: img is now Image component

### LINKED ISSUE #259: img tag needs aria labels
There's some errors related to eslint. See https://app.netlify.com/sites/opensauced-landing/deploys/6630576edf5afe0008bc9070#L101-L634

_Originally posted by @nickytonline in https://github.com/open-sauced/landing-page/issues/254#issuecomment-2084266946_

### PR DESCRIPTION
**Pull request by bdougie**
## Description
nick noticed there was an eslint error on #254. I took a look and found the error to unrelated to that PR and fixed it on a new PR. 

There is an `img` that should of been `Image` and was also missing aria labels.

## Related Tickets & Documents

closes #259
## Steps to QA

Run `npm run build` locally to see it succeed.

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [x] Tier 4

## [optional] Are there any post-deployment tasks we need to perform?

More errors exist that are Next warnings. I opened an issue for that. https://github.com/open-sauced/landing-page/issues/258

## [optional] What gif best describes this PR or how it makes you feel?

giphy (22)

### CODE REVIEW

File: components/sections/home-page/GitHubMock.tsx [hunk 1]
```diff
@@ -78,10 +77,12 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
               <Image alt="Doodles" src={BackgroundStroke} />
             </div>
             <div className="relative largeTablet:left-8 largeTablet:w-4/5 largeTablet:top-8">
-                <img
-                  src={topFeature?.image as unknown as string}
-                  alt="True impact to the OSS community image"
-                />
+              <Image
+                src={topFeature?.image as unknown as string}
+                alt="True impact to the OSS community image"
```
Comment (nickytonline) [inline components/sections/home-page/GitHubMock.tsx:82]: Shouldn’t have to add a role of ima as it’s already an image. The alt text should be enough but it sounds like the eslint rules currently require an aria-label as well. I can take a peek at that tomorrow, but for now, let’s get this in to fix the the invisible links.

**Review by nickytonline — COMMENTED** [review]
:shipit:

**Review by nickytonline — APPROVED** [review]

File: components/sections/home-page/GitHubMock.tsx (hunk 1 above)
Comment (nickytonline) [reply components/sections/home-page/GitHubMock.tsx:82]: Put up a small change to this after testing locally.

**Review by brandonroberts — COMMENTED** [review]
🔗

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**